### PR TITLE
Brand refresh

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

Based on @strath-m's https://www.figma.com/file/jHNCnyxgB0DVLPTwmbOohq/JBM-Brand-Refresh-Website-2023?node-id=4%3A41&t=O6BiBoUTPf3vCTbs-0 

- [x] Typograph

- [x] **Border radius's**

- [x] **Images/logo**

- [x] **Colors** (Strath finalising)

# Known TODOs:
- Button hover effects 
- Every old blue (haze) still in the UI (e.g currency switch on pay input) -> Strath finalizing 
- `/projects` search bar search icon border 

# Follow-up issues:
- JB-108 (https://linear.app/peel/issue/JB-108/make-tailwind-vars-for-fonts-font-agrandir-and-replace-inline)
- JB-109 (https://linear.app/peel/issue/JB-109/new-percent-change-and-status-on-trending-project-cards)

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by tomquirk** [comment]
Leaving notes here:
- [ ] minimize the logos as much as possible. Imageoptim + reduce dimensions
- [ ]

File: src/styles/antd-overrides/button.scss [hunk 1]
```diff
@@ -1,17 +1,18 @@
 .ant-btn {
   background: transparent;
-  @apply border-haze-400 dark:border-slate-300;
+  @apply border-bluebs-400 dark:border-slate-300;
   @apply text-over-action-secondary;
   @apply font-medium;
-  border-radius: 1px;
+  border-radius: 8px;
```
Comment (tomquirk) [inline src/styles/antd-overrides/button.scss:6]: **suggestion** lets apply a tailwind class here. if there isnt one for 8px, lets make one: https://tailwindcss.com/docs/border-radius#customizing-your-theme

File: src/styles/index.scss [hunk 2]
```diff
@@ -60,9 +103,30 @@ h6 {
...
+h1, h2 {
+  font-family: Agrandir-Wide, sans-serif;
+  @apply font-bold
+}
+
+h3,
+h4,
+h5,
+h6 {
+  font-family: Agrandir, sans-serif;
```
Comment (tomquirk) [inline src/styles/index.scss:115]: **suggestion** i think we can get a tailwind class for the fonts

https://tailwindcss.com/docs/font-family#customizing-your-theme

File: src/styles/index.scss [hunk 3]
```diff
@@ -60,9 +103,30 @@ h6 {
   @apply text-primary;
 }
 
+h1, h2 {
+  font-family: Agrandir-Wide, sans-serif;
+  @apply font-bold
+}
+
+h3,
```
Comment (tomquirk) [inline src/styles/index.scss:111]: **suggestion**: i think we should just "reset" heading styles and style each heading element inline.

https://meyerweb.com/eric/tools/css/reset/

File: src/styles/index.scss [hunk 4]
```diff
@@ -1,8 +1,51 @@
 @import './utilities';
 
+
+/* FONT FACES */
```
Comment (tomquirk) [inline src/styles/index.scss:4]: **suggestion** may aswell move to their own scss file and import them

File: tailwind.config.js [hunk 5]
```diff
@@ -17,14 +17,14 @@ module.exports = {
...
-      haze: {
+      bluebs: {
         25: '#F5FEFF',
         50: '#E9F9FA',
         100: '#D7F6FA',
         200: '#AAEAF2',
         300: '#48D0E0',
-        400: '#18B4C7',
-        500: '#049EB2',
+        400: '#5777EB',
```
Comment (wraeth-eth) [inline tailwind.config.js:26]: Is this a mistake? (400 and 500 are the same)

**Comment by wraeth-eth** [comment]
Is it possible to move the fonts to public assets?

**Review by wraeth-eth — COMMENTED** [review]
fresh as f*ck! ❤️

**Comment by wraeth-eth** [comment]
image

Connect still mono dm

**Review by tomquirk — APPROVED** [review]

**Pull request merged**